datasets/cn/national_congress/crawler.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+import csv
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+
+def crawl_item(input_dict: dict, context: Context):
+    name = input_dict.pop("姓名/Name")
+    ethnicity = input_dict.pop("民族/Ethnicity")
+    gender = input_dict.pop("性别/Gender")
+    delegation = input_dict.pop("代表团/Delegation")
+
+    entity = context.make("Person")
+    entity.id = context.make_slug(name, ethnicity, gender, delegation)
+
+    entity.add("name", name, lang="chi")
```

Comment:
I think you can also add the position column value like "Chairman of Inner Mongolia Mengcao Ecological Group" to the Person:position property. We don't use it normally any more but in this case I think it's useful a description for identifying the person.

Reply:
Ok, in this case would it be ok to write something like "Chairman of 北京" (part in English and part in Chinese)? Because in the data the name of the delegations is in Chinese.

Reply:
Sorry, I forgot I was looking at a google translate of the page.

Use the values as they are in the table.